internal/xdscache/v3/listener.go

```diff
@@ -389,8 +391,8 @@ func (c *ListenerCache) OnChange(root *dag.DAG) {
 				NumTrustedHops(cfg.XffNumTrustedHops).
 				MaxRequestsPerConnection(cfg.MaxRequestsPerConnection).
 				Tracing(envoy_v3.TracingConfig(envoyTracingConfig(cfg.TracingConfig))).
```

Comment:
Tangentially related, looks like tracing is also in a different order between the two; for HTTP vhosts tracing comes before ExtAuth whereas for TLS vhosts it's after ExtAuth. Ideally we can make that consistent as well; @yangyy93 do you have thoughts on the preferred order for tracing?

Reply:
The [order of envoy's filters](https://www.envoyproxy.io/docs/envoy/v1.26.3/intro/arch_overview/http/http_filters#arch-overview-http-filters-ordering) is important, because envoy will process traffic according to the order of filters. It is good to process auth first, and then global rate limit. Tracing configuration takes effect in envoy and the code order in contour is irrelevant. It would be better if the sequence consistency can be maintained in http vhosts and tls vhosts.

Reply:
Ah good call, I was thinking tracing was an HTTP filter but that's not the case.